pinc/page_table.inc

```diff
@@ -548,6 +531,39 @@ function changeSelection(sel) {
     }
 }
 
+/**
+ * Can the currently logged in user see the usernames of users who have
+ * worked on a particular page in a project?
+ *
+ *
+ * @param Project $project
+ * @param array $page_res
```

Comment:
I don't love that this take the page data schema from `fetch_page_table_data()` as it seems very limited to its use with that function (see also the comment about needing this same auth data for `api_v1_project_page_round()`. Generalizing it so that it works for `api_v1_project_page_round()` too will probably fix this though.

Reply:
Yeah, I've changed it to work with a project and list of usernames.